src/Functions/parseDateTime.cpp

```diff
@@ -494,14 +494,14 @@ namespace
                     arguments[0].type->getName(),
                     getName());
 
-            if (arguments.size() > 1 && !isString(arguments[1].type))
+            if (!isString(arguments[1].type))
```

Comment:
Maybe use validateFunctionArgumentTypes helper? Like here:
https://github.com/ClickHouse/ClickHouse/blob/master/src/Functions/regexpExtract.cpp#L49-L57

Reply:
Tnx, didn't know about this class. Sounds cool, will use it.